From e950076521d8e5517edb5dbd04f1b741944a0675 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Fri, 20 Dec 2024 15:48:11 +0100
Subject: [PATCH] setup-desktop: xfce needs elogind to shutdown/reboot

Install elogind and polkit-elogind so users can shotdown and reboot.
---
 setup-desktop.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/setup-desktop.in b/setup-desktop.in
index 76d8767..f4bd263 100644
--- a/setup-desktop.in
+++ b/setup-desktop.in
@@ -69,10 +69,11 @@ case "$de" in
 		;;
 	xfce)
 		setup-xorg-base xfce4 ${BROWSER:-firefox} \
+			elogind \
 			gvfs \
 			lightdm \
 			lightdm-gtk-greeter \
-			polkit \
+			polkit-elogind \
 			xfce4-screensaver \
 			xfce4-terminal \
 			"$@"
-- 
2.47.1

